public/doc/openapi-3-v1.0.json

```diff
@@ -1386,6 +1386,9 @@
             "example": "Endpoint",
             "type": "string"
           },
+          "source_id": {
```

Comment:
should we update v1.0?

Reply:
Not sure, I thought we were supposed to but should we just only update the latest?

Reply:
I'm not sure too, theoretically yes, but practically v1.0 is used by someone so let's keep it updated